lib/src/main/java/io/ably/lib/realtime/AblyRealtime.java

```diff
@@ -168,6 +184,19 @@ public interface Channels extends ReadOnlyMap<String, Channel> {
         void release(String channelName);
     }
 
+    private LiveObjectsPlugin tryInitializeLiveObjectsPlugin() {
+        try {
+            Class<?> liveObjectsImplementation = Class.forName("io.ably.lib.objects.DefaultLiveObjectsPlugin");
+            return (LiveObjectsPlugin) liveObjectsImplementation
+                .getDeclaredConstructor(PluginConnectionAdapter.class)
+                .newInstance(this.connection.connectionManager);
+        } catch (ClassNotFoundException | InstantiationException | IllegalAccessException | NoSuchMethodException |
+                 InvocationTargetException e) {
+            Log.w(TAG, "LiveObjects plugin not found in classpath. LiveObjects functionality will not be available.", e);
```

Comment:
shouldn't be warn, should be info, it's absolutely OK to not have LiveObjects in the classpath

Reply:
Okay, will update accordingly 👍

Reply:
updated